test/prob/gamma/gamma_ccdf_log_test.hpp

```diff
@@ -11,32 +11,26 @@ class AgradCcdfLogGamma : public AgradCcdfLogTest {
                     vector<double>& ccdf_log) {
     vector<double> param(3);
 
-    param[0] = 1.0;  // y
-    param[1] = 2.0;  // alpha
-    param[2] = 2.0;  // beta
+    param[0] = 0.7;  // y
+    param[1] = 1.1;  // alpha
+    param[2] = 0.9;  // beta
```

Comment:
Why did these change?

Reply:
I changed these, but they can change back.

If you increase the difficulty of the tests by uncommenting the `.val_.d_` stuff [here](https://github.com/stan-dev/math/blob/034d00987d4106431711ba885edfc141710486b1/test/prob/utility.hpp#L97) you'll find lots of errors (things off by 1e-14 in high order derivatives) and if you fiddle with the values here you can get them to go away.

I added those `.val_.d_` tests with this pull request. They weren't there previously.

---

Short version cause that sounded confusing: the changes to the parameter values and the changes to `.val_.d_` intializers are both new and both can be reverted.